src/assets/typography.css

```diff
@@ -0,0 +1,363 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;600;700;900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;600;700;900&display=swap');
+
```

Comment:
try mo tong idownload pra di tayo naka depende sa internet lagi same to ng ginawa ntin sa bootstrap